frontend/src/views/CreateOrderView.vue

```diff
@@ -0,0 +1,91 @@
+<script setup>
+import Button from "primevue/button";
+import Dropdown from "primevue/dropdown";
+import FloatLabel from "primevue/floatlabel";
+import InputText from "primevue/inputtext";
+import Textarea from "primevue/textarea";
+import { ref } from "vue";
+import { useRoute } from "vue-router";
+
+import { OrderStatus } from "@/global/types/appTypes";
+
+const route = useRoute();
+
+// const emit = defineEmits(["close-modal"]);
+// function onModalClose() {
+//   emit("close-modal");
+// }
+const orderStatusTypes = Object.values(OrderStatus);
+let status = ref(OrderStatus.preparation);
+
+const discountChoice = ["ja", "nein"];
+let discount = ref(discountChoice[0]);
+
+const discountPeriodChoice = ["7", "14"];
+let discountPeriod = ref(discountPeriodChoice[0]);
+
+let decription = ref();
+</script>
+<template>
+  <!-- <Dialog :visible="true" modal :closable="false" :style="{ width: '25rem' }"> -->
+  <form>
+    <div class="flex flex-col gap-y-4">
+      <p class="font-bold">Daten</p>
+      <FloatLabel>
+        <InputText id="constructionProject" class="w-full" />
+        <label for="constructionProject">Bauvorhaben</label>
+      </FloatLabel>
+      <Dropdown
+        v-model="status"
+        :options="orderStatusTypes"
+        placeholder="Status"
+        class="w-full"
+        id="select"
+      />
```

Comment:
Can we add a label here? With a default status already it is hard to know what this dropdown signifies